doc/source/migration-guide.rst

```diff
@@ -315,7 +315,7 @@ Here, you will first check the correct folder structure. If the project structur
 2.5. Move rest of text files
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Files showing as (2) "deleted" upon git status are in the main repo but not in the scikit-package repo. We took care of most of these by moving over the src tree, but let's do the rest now. Go down the list and for <filename> in the ``git status`` "delete" files type ``cp -n ../<filepath>/<filename> ./<target_filepath>``. Do not move files that we do not want. If you are unsure, please confirm with Project Owner.
+#. Files showing as (2) "deleted" upon git status are in the main repo but not in the scikit-package repo. We took care of most of these by moving over the src tree, but let's do the rest now. Go down the list and for <filename> in the ``git status`` "delete" files type ``cp -n ../<filepath>/<filename> ./<target_filepath>``. Do not move files that we do not want. If you are unsure, please confirm with Project maintainer.
```

Comment:
this could be owner, or owner or maintainer here.

Reply:
done 

_please confirm with the project owner or maintainer(s)._